apps/condo/domains/notification/schema/PushToken.js

```diff
@@ -0,0 +1,72 @@
+/**
+ * Generated by `createschema notification.PushToken PushToken`
+ */
+
+const { Text, Relationship, Select } = require('@keystonejs/fields')
+
+const { GQLListSchema } = require('@core/keystone/schema')
+const { historical, versioned, uuided, tracked, softDeleted } = require('@core/keystone/plugins')
+
+const { SENDER_FIELD, DV_FIELD } = require('@condo/domains/common/schema/fields')
+const access = require('@condo/domains/notification/access/PushToken')
+
+const { DEVICE_SERVICE_TYPES_KEYS } = require('../constants')
+
+
+const PushToken = new GQLListSchema('PushToken', {
+    schemaDoc: 'Push token',
+    fields: {
+        dv: DV_FIELD,
+        sender: SENDER_FIELD,
+
+        user: {
```

Comment:
owner?

Reply:
Renamed. By the way I totally didn't like migrations that were created automatically. Instead of renaming field it's being totally removed and created new one, loosing all previous data. I'll need some help with that, how to correctly fix this problem, my idea is to rename field in initial migration and remove second one at all, but I don't know if that could affect some migration counters...